app/src/views/CookiePolicy/i18n.json

```diff
@@ -0,0 +1,70 @@
+        "dataAccessSharingDescription3": "Further, your data will only be accessible to those staff (and other authorized individuals) that have a legitimate need to have such access.",
+        "storageSecurityQuestionsAboutDataHeading": "D) Storage ,security, questions about your data",
+        "storageSecurityQuestionsDataDescription1": "The Federation uses secure systems and implements appropriate technical and organizational measures to safeguard your personal information and protect it against unauthorized or unlawful processing, accidental loss, destruction, or damage. While we take reasonable measures to protect your personal information, please be aware that no system can guarantee 100% security.",
+        "storageSecurityQuestionsDataDescription2": "Where it is necessary for your personal data to be shared with third parties, we will take reasonable measures to ensure that your personal data continues to be treated securely.",
+        "storageSecurityQuestionsAboutDataDescription3": "These measures may be specific contractual provisions requiring those third parties to implement no less equivalent security measures than those of the Federation. Please be aware of this when sending any communications. We cannot (and do not) guarantee absolute security. Under certain circumstances, you may request to review personal information we hold about you. You may also object to certain personal data processing and/or request the correction or deletion of such personal data according to our {termsLink}. More detail about data subject requests can be found in Section 3 of that Policy.",
+        "policyProtectionOfPersonalDataLink": "Policy on the  Protection of Personal Data",
+        "storageSecurityQuestionsAboutDataDescription4": "For further information on how the IFRC processes personal data, please consult our {termsLink}. Here you may also find general information, useful tools for humanitarians, and information specific to procurements and other partners of the Federation.",
+        "dataProtectionPageLink": "Data Protection page",
+        "storageSecurityQuestionsDataDescription5": "Please contact the following department for personal data questions related to:",
+        "storageSecurityQuestionsDataGoEnquires": "IFRC-GO enquiries: {termsLink}",
+        "ifrcGoEnquiresLink": "im@ifrc.org",
+        "storageSecurityQuestionsDataDonations": "Donations:{termsLink}",
+        "donationsLink": "prd@ifrc.org",
+        "storageSecurityQuestionsDataRecruitment": "Recruitment:{termsLink}",
+        "recruitmentLink": "ask.hr@ifrc.org",
+        "securityQuestionsWebpageCollection": "General webpage collection:{termsLink}",
+        "webpageCollectionLink": "webteam@ifrc.org",
+        "storageSecurityQuestionsDataEnquires": "Other data protection enquiries:{termsLink}",
+        "dataProtectionLink": "dataprotection@ifrc.org",
+        "privilegesAndImmunitiesHeading": "E) Privileges and immunities",
```

Comment:
use mailTo for this email links and if its already done then remove these unused strings.